lcd_test/helpers.go

```diff
@@ -369,19 +369,19 @@ func init() {
 
 // CreateAddr adds an address to the key store and returns an address and seed.
 // It also requires that the key could be created.
-func CreateAddr(name, password string, kb crkeys.Keybase) (sdk.AccAddress, string, error) {
+func CreateAddr(name string, kb crkeys.Keybase) (sdk.AccAddress, string, error) {
 	var (
 		err  error
 		info crkeys.Info
 		seed string
 	)
-	info, seed, err = kb.CreateMnemonic(name, crkeys.English, password, crkeys.Secp256k1)
+	info, seed, err = kb.CreateMnemonic(name, crkeys.English, client.DefaultKeyPass, crkeys.Secp256k1)
```

Comment:
why the change?

Reply:
`CreateAddr()` doesn't take `password` anymore. Thus I fell back to the default password (it could be even an empty string)